test/parallel/test-symlinked-peer-modules.js

```diff
@@ -0,0 +1,78 @@
+// Flags: --preserve-symlinks
+/* eslint no-irregular-whitespace: 0 */
```

Comment:
Literally just cut n pasted the tree from the terminal tree command output.
I believe it contains some non-typical whitespace. No problem removing it,
just didn't seem all that important to do so.
On May 12, 2016 3:21 AM, "Ben Noordhuis" <notifications@github.com> wrote:

> In test/parallel/test-symlinked-peer-modules.js
> <https://github.com/nodejs/node/pull/6537#discussion_r62997320>:
>
> > @@ -0,0 +1,78 @@
> > +// Flags: --preserve-symlinks
> > +/* eslint no-irregular-whitespace: 0 */
>
> Why is no-irregular-whitespace: 0 necessary? Because of the ASCII diagram
> below?
>
> (It's not really ASCII but you get my meaning.)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6537/files/ad138be3ec882a3370cd8ab1b1902264b8c2ccda#r62997320>
>